### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by ar4s**
dashboard

### CODE REVIEW

File: src/ralph/admin/templatetags/dashboard_tags.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            'color': next(color),
+            'dc': name,
+            'capacity': int(capacity)
+        })
+    return {'capacities': results}
+
+
+@register.inclusion_tag('admin/templatetags/ralph_summary.html')
+def ralph_summary():
+    items = [
```
Comment (ar4s) [inline src/ralph/admin/templatetags/dashboard_tags.py]: - [x] refactor

File: gulpfile.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ var config = {
 }
 
 var sass_config = {
-    outputStyle: 'compressed',
+    // outputStyle: 'compressed',
```
Comment (mkurek) [inline gulpfile.js]: ?

**Comment by vi4m** [comment]
Awesome :-)

**Comment by fossilet** [comment]
👍

File: src/ralph/admin/templatetags/dashboard_tags.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    difference = dict(available_space - used_space)
+    results = []
+    for name, value in sorted(difference.items()):
+        capacity = 100 - (100 * value / available_space[name])
+        tooltip = '<strong>Free U:</strong> {} ({} in total)'.format(
+            available_space[name] - int(used_space[name]),
+            available_space[name]
+        )
+        results.append({
+            'url': '/dc_view#/dc/{}'.format(data_centers_mapper[name]),
```
Comment (mkurek) [inline src/ralph/admin/templatetags/dashboard_tags.py]: reversion?

**Comment by szok** [comment]
:ok:
Remove redirect to BO Asset after login

**Comment by mkurek** [comment]
:ok:

**Pull request merged**